docs/root/intro/version_history.rst

```diff
@@ -32,7 +32,7 @@ Bugfixes:
 
 Features:
 
-- Android & iOS: add support for registering a platform KV store (:issue: `#2134 <2134>`) (:issue: `#2335 <2335>`)
+- Android, iOS, & C++: add support for registering a platform KV store (:issue: `#2134 <2134>`, :issue: `#2335 <2335>`, :issue: `#2430 <2430>`)
```

Comment:
Should we get into a habit of referring to the various APIs by language instead of a mix of platform and language?

`Android, iOS, & C++` -> `Kotlin, Swift & C++ `

Reply:
No strong feelings. This labeling is purely historical, and we've just stuck with it to be consistent. To your point, the library itself is organized by language. The only point I can think of in favor of the platform names is that it perhaps highlights that the library is intended for use in these environments to the unfamiliar user. But the README can do that, too.

I'm good either way.